mantis-control-plane/mantis-control-plane-server/build.gradle

```diff
@@ -62,6 +63,43 @@ dependencies {
     testImplementation testFixtures(project(":mantis-control-plane:mantis-control-plane-core"))
     testImplementation libraries.commonsIo
 }
+def installDir = file("${buildDir}/install")
+def controlPlaneState = file("${project.resources}/docker")
+def ci = System.getenv('GITHUB_ACTIONS')
+def imageRepository = ci ? 'netflixoss' : 'localhost:5001/netflixoss'
+
+docker {
+    dockerSyncBuildContext {
+        from installDir
+        from controlPlaneState
+        from('docker') {
+            into 'docker'
+        }
+    }
+
+    dockerCreateDockerfile {
+        instruction 'RUN apt-get update && apt-get install -y curl'
+
+        // instructions to install all the necessary dependencies
+        instruction 'WORKDIR /apps/mantis/mantis-control-plane-server'
+        instruction 'COPY mantis-control-plane-server/bin/* bin/'
+        instruction 'COPY mantis-control-plane-server/lib/* lib/'
+        instruction 'COPY resources/* conf/'
+        instruction 'COPY docker/job-clusters/* job-clusters/'
+        instruction 'COPY docker/scripts/* bin/'
+
+        entryPoint './bin/start.sh'
+    }
+
+    javaApplication {
```

Comment:
`javaApplication` will create an entryPoint in the generated Dockerfile. And then the previous `dockerCreateDockerfile` section will also create an entryPoint. is that OK? I'm not sure if the generated Dockerfile is actually valid.

Reply:
Yeah, the second entry point overrides the first one. I have tested it end to end, and I can give a demo if you would like.